### DISCUSSION TITLE
A question about how custom constraints are accounted for in the score function.

### DISCUSSION THREAD

**Discussion by tsamdin** [RosettaCommons/rosetta#113] [General]
If I have an atompair constraint, added using the ConstraintSetMover, and I want to increase the weight of these constraints as rosetta designs/packs do I just need to reweight the scoretype="atom_pair_constraint" value in my scoring function?

**Comment by gfzhou** [comment]
You can use `-set_weights` flag in command line to change the weight of any score term, e.g. `-set_weights atom_pair_constraint 2.0` is setting the weight to 2.0 for `atom_pair_constraint`.

**Reply by tsamdin** [reply]
Thank you!

**Comment by roccomoretti** [comment]
If you're using RosettaScripts, a more flexible approach is indeed to modify the ScoreFunction you're using in the ScoreFunctions block by using the [Reweight subtag](https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/RosettaScripts#rosettascript-sections_scorefunctions).

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
